_maps/RandomRuins/SpaceRuins/hellfactory.dmm

```diff
@@ -728,10 +728,7 @@
 /area/ruin/space/has_grav/hellfactory)
 "ct" = (
 /obj/machinery/light/built/directional/south,
-/obj/structure/marker_beacon{
-	icon_state = "markerburgundy-on"
-	},
-/turf/open/floor/iron,
+/obj/structure/marker_beacon/burgundy/turf/open/floor/iron,
```

Comment:
mr "i NEVER text edit my maps no sir"

Reply:
i was more upset about the web edit not the text edit